inert/dynamic-inert-on-focused-element.tentative.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Dynamic inertness on focused element</title>
+<link rel="author" title="Oriol Brufau" href="mailto:obrufau@igalia.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/interaction.html#focus-fixup-rule">
+<meta name="assert" content="If a focused element becomes expressly inert, it stops being focused.">
+<div id="log"></div>
+
+<input class="has-focus becomes-inert">
+
+<div class="becomes-inert">
+  <input class="has-focus">
+</div>
+
+<button class="has-focus becomes-inert">foo</button>
+
+<div class="has-focus becomes-inert" tabindex="-1"></div>
+
+<div class="becomes-inert">
+  <div class="has-focus" tabindex="-1">bar</div>
+</div>
+
+<div class="becomes-inert">
+  <p>
+    <span class="has-focus" tabindex="-1">baz</span>
+  </p>
+</div>
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+function nextRepaint() {
+  return new Promise(resolve => {
+    requestAnimationFrame(() => {
+      requestAnimationFrame(resolve);
+    });
+  });
+}
+
+promise_setup(nextRepaint);
```

Comment:
what about waiting for page load?

Reply:
The reason for this is that the first test was flaky in old chromium: it would pass if the test started before the first painting.
Awaiting load seems to work too, but it's more verbose if I'm already using `nextRepaint`